src/model/src/ASX_DATA_MOD.F

```diff
@@ -261,13 +251,9 @@ Module ASX_DATA_MOD
       Real, allocatable, private :: BUFF2D( :,: )   ! 2D temp var
       Real, allocatable, private :: BUFF3D( :,:,: ) ! 3D temp var
 
-! Canopy option control (Wei Li)
```

Comment:
@btang1 Please do not remove any of my canopy mods to the ASX_DATA_MOD.F.  Please revert to include all my canopy changes in updated ASX_DATA_MOD.

Reply:
let me double check please

Reply:
I have made updates, move back all canopy inline for ASX_DATA_MOD